packages/utils/upgrade/src/tasks/upgrade/requirements/minor.ts

```diff
@@ -0,0 +1,21 @@
+import * as semver from 'semver';
+import { requirementFactory } from '../../../modules/requirement';
+
+export const REQUIRE_LATEST_FOR_CURRENT_MINOR = requirementFactory(
+  'REQUIRE_LATEST_FOR_CURRENT_MINOR',
+  (context) => {
+    const { project, npmVersionsMatches } = context;
+    if (npmVersionsMatches.length !== 1) {
+      const invalidVersions = npmVersionsMatches.slice(0, -1);
+      const invalidVersionsAsSemVer = invalidVersions
+        .filter((v) => semver.minor(v.version) === project.strapiVersion.minor)
+        .map((v) => v.version);
+      const nbInvalidVersions = invalidVersionsAsSemVer.length;
+      if (nbInvalidVersions !== 0) {
+        throw new Error(
+          `Doing a minor upgrade requires to be on the latest patch for v${project.strapiVersion} version, but found ${nbInvalidVersions} patch(es) ${invalidVersionsAsSemVer}`
```

Comment:
Does it though? I believed you could just run a minor upgrade without having to run a patch and then a minor to accommodate both bumps directly

Also, I don't see this requirement in the feature specification, did I miss something regarding this?

Reply:
Not sure maybe i understood the user story incorrectly